packages/react-reconciler/src/Scheduler.js

```diff
@@ -26,3 +26,6 @@ export const NormalPriority = Scheduler.unstable_NormalPriority;
 export const LowPriority = Scheduler.unstable_LowPriority;
 export const IdlePriority = Scheduler.unstable_IdlePriority;
 export type SchedulerCallback = (isSync: boolean) => SchedulerCallback | null;
+
+export const unstable_yieldValue = Scheduler.unstable_yieldValue;
+export const _setDisableYieldValue = Scheduler._setDisableYieldValue;
```

Comment:
I don't think I've seen us export something with a name beginning with "_" anywhere before. I _think_ we just want this behavior for testing (aka the `SchedulerMock`) but we don't need it for the "real" scheduler? If that's the case, couldn't we just add this export to the mock scheduler and not the other ones? (We already have a typeof check anyway before calling the scheduler methods.)

Reply:
@acdlite recommended that we add this to the Scheduler file. Open to also just directly importing `scheduler-mock` if you all think it's a better solution though!

Reply:
Gotcha 👍🏼 Thanks for renaming! :)